lib/filterStorage.js

```diff
@@ -615,7 +615,7 @@ let FilterStorage = exports.FilterStorage =
   /**
    * @typedef FileInfo
    * @type {object}
-   * @property {nsIFile} file
+   * @property {number} index
```

Comment:
This change and `package-lock.json` are unrelated and should not be here, could you please fix the pull request in that regard.

Reply:
Seems you missed the comment about this change? (If not my appologies, the GitHub codereview tool isn't the best!)

Reply:
I misunderstood. I removed the package-lock.json, thought that's what you meant. Do you want me to bring back this file to how it was before I changed it?

Reply:
I don't have a clue why it appears here, this file was not touched in this pull request and wasnt added or commited. Let me know what I should do to fix this...

Reply:
Well the changes are both in this commit https://github.com/jaels/adblockpluscore/commit/8ed57c914c89ccac9eae65e00b76b32c61b7d307 . You could simply revert that commit but I think the proper solution would be to use feature branches for your development work.

I'd point your master branch at the same commit as adblockplus/adblockpluscore (https://github.com/adblockplus/adblockpluscore/commit/78a11e759e56c29dd4e12b8ad2d601ceb84c12ec) and then I'd create a new branch called `1846-remove-locale-param` or similar which is branched from master but with your changes for this pull request placed on top. Then let's say you start working on another issue you can check out the master branch again, make sure it's up to date before creating another feature branch off that.